site/tutorials/gettings_started.md

```diff
@@ -315,3 +321,5 @@ Now you can create a website that embeds a Vega-Lite specification. If you want
 - See the [examples gallery]({{site.baseurl}}/examples/gallery.html).
 - Build your own visualizations in the [online editor](https://vega.github.io/vega-editor/?mode=vega-lite).
 - Browse through the [documentation]({{site.baseurl}}/docs/).
+- Learn [how to manually compile Vega-Lite specifications into Vega specifications](https://vega.github.io/vega-lite/usage/compile.html).
```

Comment:
In the context of this tutorial, yes. It's an advanced topic that has it's place in a later tutorial.